modules/repository-datastax/integration-test/test/java/com/intuit/wasabi/repository/cassandra/impl/authorization/AuthorizationRepositorySetup.java

```diff
@@ -22,34 +37,40 @@
 @Guice(modules = CassandraRepositoryModule.class)
 public class AuthorizationRepositorySetup {
     private final Logger logger = LoggerFactory.getLogger(AuthorizationRepositorySetup.class);
-    @Inject MappingManager mappingManager;
-    @Inject AppRoleAccessor appRoleAccessor;
-    @Inject UserInfoAccessor userInfoAccessor;
-    @Inject UserRoleAccessor userRoleAccessor;
-    @Inject ApplicationListAccessor applicationListAccessor;
-    @Inject UserDirectory userDirectory;
+    @Inject
+    MappingManager mappingManager;
+    @Inject
+    AppRoleAccessor appRoleAccessor;
+    @Inject
+    UserInfoAccessor userInfoAccessor;
+    @Inject
+    UserRoleAccessor userRoleAccessor;
+    @Inject
+    ApplicationListAccessor applicationListAccessor;
+    @Inject
+    UserDirectory userDirectory;
 
     protected final Set<String> apps = new HashSet<>();
     protected final Map<String, Set<String>> appUser = new HashMap<>();
 
-    public void setupDb(String appName, String username, String role){
+    public void setupDb(String appName, String username, String role) {
         UserInfo userInfo = userDirectory.lookupUser(UserInfo.Username.valueOf(username));
-        logger.info(appName+" "+" "+username+" "+role + " "+userInfo);
-        if(!"*".equals(appName)){
+        logger.info(appName + " " + " " + username + " " + role + " " + userInfo);
+        if (!"*".equals(appName)) {
```

Comment:
@mans4singh those are all good comments we should keep in mind, but let's do only the headers here. Will update the PR shortly